x-pack/solutions/search/plugins/enterprise_search/public/navigation_tree.ts

```diff
@@ -104,6 +104,18 @@ export const getNavigationTreeDefinition = ({
                   },
                   link: 'dashboards',
                 },
+                {
+                  children: [
```

Comment:
FYSA, whenever this is ungated there are navigation FTRs that will need to be updated.

Reply:
good point! I will add a ticket to track this. But hopefully once we ungate chat, FTRs will fail on PRs CI?